### PR TITLE
Map async clipboard tests to web-features

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: clipboard-apis/WEB_FEATURES.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+features:
+- name: async-clipboard
+  files: async-*
```
[the remark this answers is not in the document]
Comment (jcscottiii) [reply clipboard-apis/WEB_FEATURES.yml:3]: Currently wildcards are not supported. But let me try to add that to my experiment branch. It shouldn't be too hard.

When it is supported, I would expect this file to look like:

```
features:
- name: async-clipboard
  files:
  - async-*
```

[the remark this answers is not in the document]
Comment (foolip) [reply clipboard-apis/WEB_FEATURES.yml:3]: Aha, so we need to use a list for files except when it's the special value '**', right?

[the remark this answers is not in the document]
Comment (jcscottiii) [reply clipboard-apis/WEB_FEATURES.yml:3]: That's correct!

[the remark this answers is not in the document]
Comment (foolip) [reply clipboard-apis/WEB_FEATURES.yml:3]: https://github.com/web-platform-tests/wpt/pull/43403 will make the `async-*` pattern here work, right?

[the remark this answers is not in the document]
Comment (jcscottiii) [reply clipboard-apis/WEB_FEATURES.yml:3]: That is correct!

**Review by jcscottiii — APPROVED** [review]

**Comment by foolip** [comment]
Taking a closer look at https://wpt.fyi/results/clipboard-apis, actually many more tests are for the async API. This is a case where excluding tests that _aren't_ part of the feature would be easier.

Nevertheless, I'll update this PR to show what it would look like to map all of the tests with the current mechanism.

**Comment by foolip** [comment]
Actually, it turned out kind of reasonable, I don't think we need an exclude mechanism for this feature :D

**Pull request merged**